content-src/components/Search/_Search.scss

```diff
@@ -64,14 +64,14 @@ $glyph-forward: url('chrome://browser/skin/forward.svg');
     height: $search-height;
     margin: 0 auto;
     position: relative;
-    width: $searchbar-width-small;
```

Comment:
These changes just reducing complexity because these variables are only used in 1 place? Or is there something else you're trying to adjust, where the values in the variables are off?

Reply:
This is now removed...wasn't necessary as it was a single use var.